tests/Integration/DataFixtures/ORM/resources/ProductVariant/1111111186.yaml

```diff
@@ -0,0 +1,4 @@
+Sylius\Component\Core\Model\ProductVariant:
```

Comment:
This fixture doesn't seems to be used

Reply:
Done https://github.com/webgriffe/SyliusAkeneoPlugin/commit/328efd3840bf34d87235b61502fb7af600297f40